week-3/todo-list/script.js

```diff
@@ -1,25 +1,108 @@
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
-}
+  list.innerHTML = "";
+
+  todos.forEach((item) => {
+    const listItem = document.createElement("li");
+    list.appendChild(listItem);
+    listItem.textContent = item.task;
+    // Create and append a checkbox
+    const checkBox = document.createElement("i");
+    listItem.appendChild(checkBox);
+    checkBox.classList.add("check-box");
+
+    const span = document.createElement("span");
+    listItem.appendChild(span);
+    span.classList.add("badge", "bg-primary", "rounded-pill");
+
+    // Create and append a check icon
+    // const checkIcon = document.createElement("i");
+    // checkIcon.classList.add("fa", "fa-check");
+    // span.appendChild(checkIcon);
 
+    const crashIcon = document.createElement("i");
+    crashIcon.classList.add("fa", "fa-trash");
+    span.appendChild(crashIcon);
+
+    crashIcon.addEventListener("click", () => {
+      listItem.remove();
+      let index = todos.indexOf(item);
+      todos.splice(index, 1);
+    });
+
+    // Add a change event listener to the checkbox
+    checkBox.addEventListener("click", () => {
+      if (item.completed === false) {
+        listItem.classList.toggle("completed");
+        checkBox.classList.toggle("fa-solid");
+        checkBox.classList.toggle("fa-check");
+      }
+
+      item.completed = !item.completed;
+      // if (item.completed === false) {
```

Comment:
The PR should not contain any commented out code. If the code is no longer used, it should be deleted.